tests/Repository/EmailTemplateTest.php

```diff
@@ -30,10 +40,8 @@ protected function setUp(): void
         $this->emailTemplateRepository = $container->get('bitbag_sylius_mail_template_plugin.custom_repository.email_template');
         $this->emailCodesProvider = $container->get('bitbag_sylius_mail_template_plugin.provider.email_codes');
 
-        $loader = $container->get('fidry_alice_data_fixtures.loader.doctrine');
-        $loader->load([
-            __DIR__ . '/DataFixtures/ORM/email_templates.yml',
-            __DIR__ . '/DataFixtures/ORM/email_templates_translation.yml',
+        $this->loadFixturesFromFiles([
+             'email_templates.yml',
```

Comment:
Where is the fixtures file placed? I don't see it

Reply:
https://github.com/BitBagCommerce/SyliusMailTemplatePlugin/pull/19/commits/9930e467b8a474c346f21213eb822b9d0efe747f#diff-990cfc9c8a789541c2174347e4357eacb98088200dc829abb1c5204aa8acb0fa